website/src/data/highlights.tsx

```diff
@@ -0,0 +1,102 @@
+import Translate, { translate } from "@docusaurus/Translate";
+import React from "react";
+import SnippetAsync from "!!raw-loader!/static/snippets/async.dart";
+import SnippetCombine from "!!raw-loader!/static/snippets/combine.dart";
+import SnippetDeclare from "!!raw-loader!/static/snippets/declare.dart";
+import { IHighlightProps } from "../components/Highlight";
+
+export const highlights: IHighlightProps[] = [
+  {
+    snippet: SnippetDeclare,
+    title: translate({
+      id: "home.shared_state_title",
+      message: "Declare shared state from anywhere",
+    }),
+
+    description: (
+      <Translate
+        id="home.shared_state_body"
+        description="The homepage input placeholder"
+        values={{
+          main: <code>main.dart</code>,
+          br: <br></br>,
+        }}
+      >
+        {`No need to jump between your {main} and your UI files
+          anymore. {br}
+          Place the code of your shared state where it belongs, be
+          it in a separate package or right next to the Widget that
+          needs it, without losing testability.`}
+      </Translate>
+    ),
+  },
+  {
+    snippet: SnippetCombine,
+    title: translate({
+      id: "home.recompute_title",
+      message: "Recompute states/rebuild UI only when needed",
+    }),
+    description: (
+      <Translate
+        id="home.recompute_body"
+        values={{
+          br: <br></br>,
+          Provider: (
+            <code>
+              <a href={"./docs/concepts/combining_providers"}>Provider</a>
+            </code>
+          ),
+          families: <a href={"./docs/concepts/modifiers/family"}>"families"</a>,
+          build: <code>build</code>,
+          truly: <strong>truly</strong>,
+        }}
+      >
+        {`We no-longer have to sort/filter lists inside the {build}
+          method or have to resort to advanced cache mechanism. {br} {br}
+          With {Provider} and {families}, sort your lists or do HTTP
```

Comment:
The example doesn't have anything to do with families. I think the wording should be changed here, but I think an example involving a common usage of families would be useful.